doc/DOWNLOADS.md

```diff
@@ -128,3 +118,47 @@ A stable macOS installer for Nicotine+ is available on macOS Catalina 10.15 and
 *NOTE: You have to follow [these instructions](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac) the first time you open Nicotine+ on macOS.*
 
 - [Download Nicotine+ macOS Installer](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/macos-installer.zip)  [[SHA256](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/macos-installer.zip.sha256)]
+
+## Cross-Platform
+
+### pip
+
+Running Nicotine+ using pip requires a [MinGW environment](https://en.wikipedia.org/wiki/MinGW) on Windows and [Homebrew](https://brew.sh/) on macOS. Note that problems encountered using this method on OSes other than GNU/Linux variants are unlikely to receive support.
```

Comment:
Change the order of the words, because at first glace it appears that MinGW is required in any case on all platforms. Move the entire paragraph below the code block (on Line 134).

"Installing Nicotine+ using pip on Windows requires a MinGW environment. On macOS, this requires Homebrew. In other words, a Python installation from Python's website will not work, therefore, you should use a supported Nicotine+ installer for [Windows] or [macOS]."

Cannot use non-authoritative source such as Wikipedia. Either remove the URL, or change to something better. Suggestions for alternative links:

+ https://osdn.net/projects/mingw/

+ https://wiki.gnome.org/Projects/GTK/Win32/MinGW

+ https://wiki.gnome.org/Building%20cross-platform%20GTK%2B%20applications%20with%20MinGW%20and%20C/C%2B%2B

Reply:
@slook In case it went by unnoticed, the link I chose for the MinGW environment was not one of your suggestions. From my research, MinGW-w64 is the current better choice on Windows and so I linked that instead.